src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  @include dimensions;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+  margin-right: 4px;
```

Comment:
The last child of '.stars__star' will also have a 'margin-right'. You might want to remove it to keep the content aligned properly. You can use ':not(:last-child)' to select all elements except the last one.